### ISSUE TITLE
character conversions, e.g gt/sme/facta/RIT* (Bugzilla Bug 258)

### ISSUE THREAD

**Issue by albbas** [divvun/CorpusTools#31]
This issue was created automatically with bugzilla2github
# Bugzilla Bug 258
Date: 2006-03-14T00:53:33+01:00
From: @Trondtr (Originally created by @Trondtr)
To:   @albbas
CC:   saara.huhmarniemi

Last updated: 2015-03-16T14:16:01+01:00

**Comment by albbas** [comment]
## Comment 860
Date: 2006-03-14 00:53:33 +0100
From: @Trondtr

á=á, č=ū, đ=đ, ŋ=ņ, ŋ=ñ, š=ķ, š=ó, ž=į

**Comment by albbas** [comment]
## Comment 932
Date: 2006-05-13 13:45:23 +0200
From: Saara Huhmarniemi &lt;saara.huhmarniemi&gt;

> á=á, đ=đ, ŋ=ñ, š=ó
> 
These errors are now fixed, I added the code table winsam to our tables. (I wonder why it was missing..) It gives the following substitutions:

Đ Ð
ŧ ý  
Ŋ Ñ
đ ð
Ž Þ
Č Ç
ŋ ñ
Ŧ Ý
č ç
š ó
Š Ó
ž þ

These characters I did not find from our corpus (exept some of them in the broken title fields)
č=ū, ŋ=ņ, ž=į,  š=ķ,

I leave the bug open until the fix is verified.

**Comment by albbas** [comment]
## Comment 1003
Date: 2006-06-08 16:30:33 +0200
From: Saara Huhmarniemi &lt;saara.huhmarniemi&gt;

I am going through character conversions once more. couple of things.

There are couple of digraph-representations still left in some documents. I'll go through the text documents with c1,d1,z1,.. The strategy is to move the original files to utf-8 in the corpus database. Generally, we want to keep the original files as such, but since there files were already corrupted, I think cleaning the originals is the best fix. 

If I have understood correctly, after we have cleaned our files, the incoming files rarely contain digraph representations.

corp/orig/sme/admin/others/uito-ohpenplana.txt
corp/orig/sme/laws/nac1-1994-24.txt
corp/orig/sme/laws/nac1-1993-34.txt
corp/orig/sme/facta/regnor.txt

In other words, these are now fixed. However, the file corp/orig/sme/laws/nac1-1993-34.txt is broken due to other reasons, it would be nice to get the original file instead.

Some of the files contain different encodings. These are not taken into account at the moment. The difficulty is that small pieces are not enough to guess the correct encoding. Converting the whole document in those cases leads to errors in already correctly encoded parts of the text. Perhaps we have to consider leaving these files unconverted (they are not so many).

There are some other improvements as well, evaluation follows..
